HW_1/Task1.py

```diff
@@ -0,0 +1,19 @@
+# Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+num = int(input('Введите трёхзначное число: '))
+sum = 0
+if 99 < num <= 999:
+
+    result = num % 10
+    sum = sum + result
+    n = num//10
+    n = n % 10
+    m = num//10
+    m = m//10
+    print(sum + n + m)
```

Comment:
Не очень оптимальное решение. Разрешите, я поправлю. Внимательно посмотрите, что я сделаю.
```suggestion
if 99 < num <= 999:
    n = num % 10
    m = num // 10 % 10
    k = num // 100
    print( n + m + k)
```